runtime/doc/msgpack_rpc.txt

```diff
@@ -17,22 +17,22 @@ The Msgpack-RPC Interface to Nvim				  *msgpack-rpc*
 ==============================================================================
 1. Introduction						    *msgpack-rpc-intro*
 
-The primary means of controlling a running Nvim instance is through
+The primary way to control a running Nvim instance is through
 MessagePack-RPC, a messaging protocol that uses the MessagePack serialization
 format: https://github.com/msgpack/msgpack/blob/7498cf3/spec.md.
-From now on, we'll be referring to the protocol as msgpack-rpc.
+From now on, we refer to the protocol as msgpack-rpc.
 
 At this point, only plugins use msgpack-rpc, but eventually even user
-interaction will be achieved through the protocol, since user interfaces will
-be separate programs that control a headless Nvim instance.
+interaction will happen through the protocol, since user interfaces will be
```

Comment:
maybe `... even user interaction will happen through it, ... `? "the protocol" seems redundant since it's just been mentioned.